src/pages/PoolDetails/icons.tsx

```diff
@@ -0,0 +1,15 @@
+type SVGProps = React.SVGProps<SVGSVGElement> & {
+  fill?: string
```

Comment:
isn't fill on svgprops already?

Reply:
or rather is fill necessary? same with gradientId never seen that

Reply:
that comes from eslint not properly understanding SVGProps https://github.com/Uniswap/interface/blob/main/src/nft/components/icons.tsx#L6C13-L6C13